scripts/copy-res.js

```diff
@@ -160,10 +160,15 @@ function genLangFile(lang, dest) {
     let translations = {};
     [reactSdkFile, riotWebFile].forEach(function(f) {
         if (fs.existsSync(f)) {
-            Object.assign(
-                translations,
-                JSON.parse(fs.readFileSync(f).toString())
-            );
+            try {
+                Object.assign(
+                    translations,
+                    JSON.parse(fs.readFileSync(f).toString())
+                );
+            } catch (e) {
+                console.error("Failed: "+f, e);
+                return;
```

Comment:
I don't think this makes any difference?

you probably want to throw a new exception?